### ISSUE TITLE
EXCEPTION_ACCESS_VIOLATION

### ISSUE THREAD

**Issue by hoangphucITJP** [neo4j/neo4j#12570]
I cannot start the database with Neo4j Desktop

### Setup
**Neo4j Version:** 4.1.1
**Operating System:** Windows 10 (64bit)
**API:** Neo4j Desktop 1.3.4

### Steps to reproduce
1. Download Ne4oj desktop
2. Migrate some data to database
	```
	787,618 nodes (3 labels)
	2,103,936 relationships (2 types)
	```
3. For some unknown reason, it crashes when I start the database later

### Error output
```
APOC couln't set a URLStreamHandlerFactory since some other tool already did this (e.g. tomcat). This means you cannot use s3:// or hdfs:// style URLs in APOC. This is caused by a limitation of the JVM which we cannot fix. 
2020-08-20 04:36:24.508+0000 INFO  Starting...
2020-08-20 04:36:24.508+0000 INFO  Starting...
2020-08-20 04:36:27.097+0000 INFO  ======== Neo4j 4.1.1 ========
2020-08-20 04:36:24.508+0000 INFO  Starting...
2020-08-20 04:36:27.097+0000 INFO  ======== Neo4j 4.1.1 ========
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ffa440e57db, pid=4420, tid=5560
#
# JRE version: OpenJDK Runtime Environment (Zulu11.35+13-CA) (11.0.5+10) (build 11.0.5+10-LTS)
# Java VM: OpenJDK 64-Bit Server VM (11.0.5+10-LTS, mixed mode, tiered, compressed oops, g1 gc, windows-amd64)
# Problematic frame:
# C  [ntdll.dll+0x157db]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# An error report file with more information is saved as:
# hs_err_pid4420.log
#
# If you would like to submit a bug report, please visit:
#   http://www.azulsystems.com/support/
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

### Error log
[hs_err_pid4420.log](https://github.com/neo4j/neo4j/files/5100717/hs_err_pid4420.log)

**Comment by PetrJanouch** [comment]
Hello, thanks for reporting this. This issue seems to have the same cause as https://github.com/neo4j/neo4j/issues/12564 . It has been fixed and the fix will be available in 4.1.2. In the meantime you can try the workaround described at the linked issue.

**Comment by PetrJanouch** [comment]
4.1.2 was released last week. Please re-test with that version. I am closing this now, but feel free to reopen this if the issue is still present with 4.1.2.
